cdap-api/src/main/java/co/cask/cdap/api/common/exceptions/HTTPResponseException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.common.exceptions;
+
+/**
+ * CDAP HTTP Response Exception
+ */
+public interface HttpResponseException {
```

Comment:
If you want it to be an interface (because you want sub-class be able to choose between inheriting from Exception or RuntimeException), then the interface shouldn't be named as `Exception`. It's more like a `HttpStatusProvider` (not a very good name.. please come up with a better one).